packages/compass-query-bar/src/utils/query.ts

```diff
@@ -89,8 +91,13 @@ export function mapQueryToFormFields(
         if (!isQueryProperty(key)) {
           return null;
         }
-        const valueAsString =
-          typeof _value === 'undefined' ? '' : stringify(_value) || '';
+        let valueAsString =
+          typeof _value === 'undefined' ? '' : toJSString(_value, 0) || '';
+
+        valueAsString = prettify(valueAsString, 'javascript-expression', {
+          trailingComma: 'none',
```

Comment:
I really wouldn't mind if we make this default in Compass, definitely would prefer it to the need to have different configurations in different parts of the app 🙂

Reply:
Agreed, going to mention in the slack channel, I'll do that in a separate pr if folks are on board. Thanks for mentioning!